AO-373 Update example .conf files of argo-ncg

eimamagi opened the pull request.

eimamagi committed:
AO-373 Update example .conf files of argo-ncg

eimamagi committed:
Remove actual token.

themiszamani commented on config/ncg.conf:
```diff
@@ -8,30 +8,20 @@
 # Some optional config
 #
-#BDII = bdii.srce.hr
-#GLITE_VERSION=3.1.0
```
what do they mean? please explaun at least  ENABLE_UNICORE_PROBES why we need it and what is myproxy
NAGIOS_SERVER = nagios.fqdn
ENABLE_UNICORE_PROBES=1
MYPROXY_SERVER = myproxy.fqdn

themiszamani commented on config/ncg.conf:
```diff
@@ -42,91 +32,77 @@ VO = dteam
-    MYPROXY_NAME=NagiosRetrieve
-    # myproxy-init argument -l
-    MYPROXY_USER=nagios
+    NAGIOS_ROLE = ngi
```
explain all different values
NAGIOS_ROLE = ngi if not ngi what else can we put
when empty hosts is 1 and why?
    INCLUDE_EMPTY_HOSTS = 0
what is enable notifications? enable nagios notifications for whom
    ENABLE_NOTIFICATIONS = 0
what is SEND_TO_DASHBOARD?
    SEND_TO_DASHBOARD = 0
what is CHECK_HOSTS?
    CHECK_HOSTS = 0

    TENANT = ARGO-TEST
robot certificates. Why do we need them 
    ROBOT_CERT=/etc/nagios/globus/robocert.pem
    ROBOT_KEY=/etc/nagios/globus/robokey.pem
  when do we set   USE_ROBOT_CERT to 1
  USE_ROBOT_CERT=1
what is SEND_TO_MSG? When do we use it
    SEND_TO_MSG = 0

themiszamani commented on config/ncg.conf:
```diff
@@ -42,91 +32,77 @@ VO = dteam
+    INCLUDE_MSG_CHECKS_SEND = 0
   </Active>
-
+  <LDAP>
```
when do we need ldap?

themiszamani commented on config/ncg.conf:
```diff
@@ -42,91 +32,77 @@ VO = dteam
-    #       | xmllint --format -
-    # SAM_VOS=OPS
+    GOCDB_ROOT_URL=https://goc.egi.eu/gocdbpi
+    ENABLE_UNICORE_PROBES=$ENABLE_UNICORE_PROBES
```
What are these values. 
    ENABLE_UNICORE_PROBES=$ENABLE_UNICORE_PROBES
    BDII_HOST=bdii.fqdn
    INCLUDE_PROXY_CHECKS = 1
    INCLUDE_MSG_CHECKS_SEND = 0

eimamagi committed:
ARGO-2212 Remove support for Alias nodes.
ARGO-2211 Remove support for LB nodes.
ARGO-2199 Investigate changes needed in NCG & Nagios to support mutliple endpoint per host.

eimamagi committed:
ARGO-2217 - Add option to use PRIMARY_KEYs from GOCDB

eimamagi committed:
Update argo-ncg.spec.

eimamagi commented:
I will reopen this one later on.
